package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Do we need this on the top level? I would prefer to not have generated files in the root.

Reply:
Yes, `package.json` needs to be at the root. 

I don't think this is a generated file. Or at least it doesn't change as the result of a `mage` command, for example.

Reply:
having package.json at the root SGTM